### PR TITLE
4.8.14 dev v7

### PR DESCRIPTION
**Pull request by ravisaurav-tarento**
Hi @karthik-tarento please review and merge the PR.

### CODE REVIEW

File: src/main/java/org/sunbird/workflow/config/Constants.java [hunk 1]
```diff
@@ -317,7 +317,7 @@ private Constants() {
 	public static final String TRANSFER_SUBJECT_LINE = "Transfer Notification: #username";
 	public static final String LAST_UPDATED_ON = "lastUpdatedOn";
 	public static final String DOMICILE_MEDIUM = "domicileMedium";
-	public static final String PINCODE = "pinCode";
+	public static final String PIN_CODE = "pincode";
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/workflow/config/Constants.java]: the value should be "pinCode" -- not "pincode" -- check the user mapping for profileDetails.employmentDetails.pinCode"

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: src/main/java/org/sunbird/workflow/config/Constants.java (hunk 1 above)
Comment (ravisaurav-tarento) [reply src/main/java/org/sunbird/workflow/config/Constants.java]: This is changes as requested.

File: src/main/java/org/sunbird/workflow/config/Constants.java [hunk 2]
```diff
@@ -317,7 +317,7 @@ private Constants() {
 	public static final String TRANSFER_SUBJECT_LINE = "Transfer Notification: #username";
 	public static final String LAST_UPDATED_ON = "lastUpdatedOn";
 	public static final String DOMICILE_MEDIUM = "domicileMedium";
-	public static final String PINCODE = "pinCode";
+	public static final String PIN_CODE = "pinCode";
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/workflow/config/Constants.java]: Do not rename... it is better to add new one.

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: src/main/java/org/sunbird/workflow/config/Constants.java (hunk 2 above)
Comment (ravisaurav-tarento) [reply src/main/java/org/sunbird/workflow/config/Constants.java]: Done as recommended @karthik-tarento

**Pull request merged**